test/integration/targets/unarchive/tasks/test_download.yml

```diff
@@ -6,19 +6,9 @@
 
 - name: Test TLS download
   block:
-  - name: Install packages to make TLS connections work on CentOS 6
-    pip:
-      name:
-        - urllib3==1.10.2
-        - ndg_httpsclient==0.4.4
-        - pyOpenSSL==16.2.0
-      state: present
-    when:
-      - ansible_facts.distribution == 'CentOS'
-      - not ansible_facts.python.has_sslcontext
   - name: unarchive a tar from an URL
     unarchive:
-      src: "https://releases.ansible.com/ansible/ansible-latest.tar.gz"
+      src: "https://ftp.gnu.org/gnu/hello/hello-1.3.tar.gz"
```

Comment:
Should we look at using one of the CI .tar.gz files in Ansible Core CI S3 bucket, for example `https://ansible-ci-files.s3.amazonaws.com/test/integration/targets/apt/echo-hello-source.tar.gz`. We can also upload this `.tar.gz` if it's legally possible.

Reply:
I will look into using the existing file. I couldn't seem to find one already used by existing tests.

Reply:
Done